spec/index.bs

```diff
@@ -2786,6 +2786,32 @@ interface GPUCommandEncoder {
 GPUCommandEncoder includes GPUObjectBase;
 </script>
 
+{{GPUCommandEncoder}} has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUCommandEncoder">
+    : <dfn>\[[state]]</dfn> of type [=encoder state=].
+    ::
+        The current state of the {{GPUCommandEncoder}}, initially set to [=encoder state/open=].
+</dl>
+
+Each {{GPUCommandEncoder}} has a current <dfn>encoder state</dfn> on the [=Content timeline=]
+which may be one of the following:
+
+ - "<dfn for="encoder state">open</dfn>" where the {{GPUCommandEncoder}} is available to begin new operations.
```

Comment:
note: at some point the encoder state might also contain "invalid" when an error happens in one of the commands. Maybe it can be deferred until more detailed error handlign is written.

Reply:
Looks resolved enough for now.